server/sonar-web/src/main/js/components/issue/BaseIssue.js

```diff
@@ -81,9 +93,20 @@ export default class BaseIssue extends React.PureComponent {
     document.removeEventListener('keypress', this.handleKeyPress);
   }
 
-  doIssueAction(action: string) {
-    this.issueView.$('.js-issue-' + action).click();
-  }
+  togglePopup = (popupName: string, open?: boolean) => {
```

Comment:
Its to allow to really toggle without having to know the current state (open if was close or close if was open).